app/config/errors.php

```diff
@@ -732,4 +742,62 @@
         'description' => 'Migration is already in progress. You can check the status of the migration in your Appwrite Console\'s "Settings" > "Migrations".',
         'code' => 409,
     ],
+
+    /** Provider Errors */
+    Exception::PROVIDER_NOT_FOUND => [
+        'name' => Exception::PROVIDER_NOT_FOUND,
+        'description' => 'Provider with the requested ID could not be found.',
+        'code' => 404,
+    ],
+    Exception::PROVIDER_ALREADY_EXISTS => [
+        'name' => Exception::PROVIDER_ALREADY_EXISTS,
+        'description' => 'Provider with the requested ID already exists.',
+        'code' => 409,
+    ],
+    Exception::PROVIDER_INCORRECT_TYPE => [
+        'name' => Exception::PROVIDER_INCORRECT_TYPE,
+        'description' => 'Provider with the requested ID is of incorrect type: ',
```

Comment:
Should this be a colon at the end? Also,  @gewenyu99 would probably review the copy for these errors.

Reply:
Yep should be, the message here should be a static fallback if it's not set in the constructor. All descriptions/docs will have their own review for sure :+1: